docs/quickstart/install.md

```diff
@@ -49,22 +62,37 @@ choco install tanzu-cli
 Note that the Chocolatey package is part of the main
 [Chocolatey community repository](https://community.chocolatey.org/packages).
 This means that when a new `tanzu-cli` version is released, the chocolatey package may
-not be available immediately.  If the above installation instructions fail, you can try:
+not be available immediately as it needs to be approved by a Chocolatey maintainer.
+If the above installation instructions install an older version of the Tanzu CLI,
+you can explicitly specify the version you want to install using the `--version` flag:
 
 ```console
 choco install tanzu-cli --version <version>
 # example: choco install tanzu-cli --version 0.90.0
 ```
 
+You can also use the `--version` flag as described above to install older
+versions of the Tanzu CLI.
+
 ### Apt (Debian/Ubuntu)
 
 ```console
-sudo apt-get update
-sudo apt-get install -y ca-certificates curl gpg
+sudo apt update
+sudo apt install -y ca-certificates curl gpg
 curl -fsSL https://packages.vmware.com/tools/keys/VMWARE-PACKAGING-GPG-RSA-KEY.pub | sudo gpg --dearmor -o /etc/apt/keyrings/tanzu-archive-keyring.gpg
 echo "deb [signed-by=/etc/apt/keyrings/tanzu-archive-keyring.gpg] https://storage.googleapis.com/tanzu-cli-os-packages/apt tanzu-cli-jessie main" | sudo tee /etc/apt/sources.list.d/tanzu.list
-sudo apt-get update
-sudo apt-get install -y tanzu-cli
```

Comment:
Was this a bug in the existing documentation?  Or is it that both `apt` and `apt-get` work?

Reply:
`apt` is the new version of `apt-get` as far as I can tell.
I had documented `apt-get` because I thought it was used probably more, but with this PR I found that `apt list` does not exist for `apt-get`, so instead of using both in the doc, I switched everything to `apt`